node_modules/col-core/lib/db.js

```diff
@@ -78,19 +78,323 @@ var init = module.exports.init = function(callback) {
+   * @property {String}     title                     The title of the asset
+   * @property {String}     description               The description of the asset
+   * @property {String}     thumbnail_url             The thumbnail url of the asset
+   * @property {String}     large_url                 The large url of the asset
+   * @property {String}     url                       The url of the asset
+   * @property {String}     source                    The source of the asset
+   * @property {String}     body                      The body of the asset
+   * @property {Number}     likes                     The number of people who liked the asset
+   * @property {Number}     dislikes                  The number of people who disliked the asset
+   * @property {Number}     views                     The number of times the assets has been viewed
+   * @property {Number}     comments                  The number of people who commented on the asset
+   */
+  var Asset = module.exports.Asset = sequelize.define('asset', {
+    'canvas_assignment_id': {
+      'type': Sequelize.INTEGER,
+      'allowNull': true
+    },
+    'type': {
+      'type': Sequelize.ENUM(assetTypes),
+      'allowNull': true
+    },
+    'title': {
+      'type': Sequelize.STRING,
+      'allowNull': true
+    },
+    'description': {
+      'type': Sequelize.TEXT,
+      'allowNull': true
+    },
+    'thumbnail_url': {
+      'type': Sequelize.STRING,
+      'allowNull': true
+    },
+    'large_url': {
+      'type': Sequelize.STRING,
+      'allowNull': true
+    },
+    'url': {
+      'type': Sequelize.STRING,
+      'allowNull': true
+    },
+    'source': {
+      'type': Sequelize.STRING,
+      'allowNull': true
+    },
+    'body': {
+      'type': Sequelize.TEXT,
+      'allowNull': true
+    },
+    'likes': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'dislikes': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'views': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'comments': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    }
+  }, {
+    'underscored': true
+  });
+
+  // Each asset belongs to a specific course
+  Asset.belongsTo(Course, {'onDelete': 'CASCADE'});
+
+  // Each asset is created by a specific user
+  Asset.belongsTo(User, {'onDelete': 'CASCADE'});
+
+  /**
+   * The `comment` table keeps track of comments that were made on assets
+   *
+   * @property  {String}        body          The body of the comment
+   */
+  var Comment = module.exports.Comment = sequelize.define('comment', {
+    'body': {
+      'type': Sequelize.TEXT,
+      'allowNull': false
+    }
+  }, {
+    'underscored': true
+  });
+
+  // Each comment is made on an asset
+  Comment.belongsTo(Asset, {'onDelete': 'CASCADE'});
+
+  // Each comment is made by a user. We do not remove comments if a user is removed from the system
+  Comment.belongsTo(User);
```

Comment:
If we don't remove a comment when a user is removed from the system, how will we be able to display that comment?